scripts/run-tests.sh

```diff
@@ -97,6 +97,15 @@ run_tests app-springboot -Pspring-boot
 # service-nodejs tests
 npm -C service-nodejs install
 npm -C service-nodejs start >/dev/null&
+# Wait for port 3000 to open for at most 30 seconds
+{
+    I=0
+    while ! echo -n > /dev/tcp/localhost/3000 && [[ $I -lt 60 ]]; do
```

Comment:
This just checks if socket is open, is there a window until it's able to process requests? Might it be better to use curl or wget to check it's ready returns a valid HTTP response?

Reply:
Yup, that's correct. Even if the window would be negligible, curl would be a bit more reliable, so I'm pushing the new version with curl